apimatic_requests_client_adapter/requests_client.py

```diff
@@ -43,7 +43,8 @@ def __init__(self,
         else:
             if override_http_client_configuration:
                 http_client_instance.timeout = timeout
-                http_client_instance.session.verify = verify
+                if hasattr(http_client_instance, 'session'):
+                    http_client_instance.session.verify = verify
                 adapters = http_client_instance.session.adapters
```

Comment:
Blocker: The adapters are being taken from session so it's better to have adapters specific statements in session attribute check.